ui/src/app.tsx

```diff
@@ -39,6 +39,7 @@ export interface RouterContext {
 // Create a new router instance
 const router = createRouter({
   routeTree,
+  basepath: import.meta.env.VITE_BASEPATH || '/',
```

Comment:
I must admit that my expertise in TypeScript/Frontend development is still developing, so I might be missing some context or best practices here. But do we need this fallback here since we are setting `UI_BASEPATH` always with a default value?

Reply:
That's only the case in Docker, when running the UI differently the `VITE_BASEPATH` variable might not be set.